### PR TITLE
Supports Color augmentation.

### PR DESCRIPTION
**Pull request by v-qjqs**
This PR implements color augmentations which only transforms image (e.g. color, equalize, sharpness) in mmcv.

### CODE REVIEW

File: mmcv/image/photometric.py [hunk 1]
```diff
@@ -91,3 +94,38 @@ def posterize(img, bits):
     shift = 8 - bits
     img = np.left_shift(np.right_shift(img, shift), shift)
     return img
+
+
+def color(img, beta, alpha=None, gamma=0, backend=None):
+    """Color the image.
```
Comment (xvjiarui) [inline mmcv/image/photometric.py:100]: Should be more detailed.

File: mmcv/image/photometric.py [hunk 2]
```diff
@@ -91,3 +93,27 @@ def posterize(img, bits):
     shift = 8 - bits
     img = np.left_shift(np.right_shift(img, shift), shift)
     return img
+
+
+def color(img, alpha=1, beta=None, gamma=0):
+    """Color the image. It blends the source image and its gray.
+
+    image with the format: `img * alpha + gray_img * beta + gamma`.
```
Comment (xvjiarui) [inline mmcv/image/photometric.py:101]:
```suggestion
    `output = img * alpha + gray_img * beta + gamma`.
```

File: tests/test_image/test_photometric.py [hunk 3]
```diff
@@ -75,3 +75,16 @@ def test_posterize(self):
...
+        img = np.array([[0, 128, 255], [1, 127, 254], [2, 129, 253]],
+                       dtype=np.uint8)
+        img = np.stack([img, img, img], axis=-1)
+        assert_array_equal(mmcv.color(img), img)
+        img_gray = mmcv.bgr2gray(img)
+        img_r = np.stack([img_gray, img_gray, img_gray], axis=-1)
+        assert_array_equal(mmcv.color(img, 0), img_r)
+        assert_array_equal(mmcv.color(img, 0, 1), img_r)
+        assert_array_equal(
+            mmcv.color(img, 0.5, 0.5),
```
Comment (xvjiarui) [inline tests/test_image/test_photometric.py:89]: We may use the value other than 0.5.

File: tests/test_image/test_photometric.py [hunk 4]
```diff
@@ -75,3 +75,16 @@ def test_posterize(self):
...
+            (img * 0.5 + img_r * 0.5).astype(img.dtype))
```
Comment (xvjiarui) [inline tests/test_image/test_photometric.py:90]: We may also add test for gamma.

**Review by xvjiarui — APPROVED** [review]

File: mmcv/image/photometric.py [hunk 5]
```diff
@@ -91,3 +93,33 @@ def posterize(img, bits):
     shift = 8 - bits
     img = np.left_shift(np.right_shift(img, shift), shift)
     return img
+
+
+def color(img, alpha=1, beta=None, gamma=0):
```
Comment (hellock) [inline mmcv/image/photometric.py:98]: The method name is too ambiguous.

**Pull request merged**